src/components/Booking/Booking.jsx

```diff
@@ -1,14 +1,24 @@
 import dayjs from "dayjs";
+import { useState } from "react";
 
 import "./Booking.scss";
 
 const Booking = (props) => {
   // prettier-ignore
   const { title, firstName, surname, email, roomId, checkInDate, checkOutDate } = props.booking;
   const stayNights = dayjs(checkOutDate).diff(dayjs(checkInDate), "d");
+  const [selected, setSelected] = useState(false);
+
+  const selectBookingHandle = () => {
+    setSelected(!selected);
+  }
 
   return (
-    <tr data-testid="booking-component">
+    <tr
```

Comment:
![image](https://github.com/HadikaMalik/React-Module-Project/assets/118762745/a982fbe1-1657-42c8-835f-db442eab3e01)
The instructions say you should pot the search results *components* into the bookings component, but instead, you wrote the search results *code* inside the booking component. Shouldnt this just have the <SearchResults /> in here and the code inside the search results component?